### PR TITLE
Allow online:int to pick up versioning after online:stg leaves off

### PR DESCRIPTION
**Pull request by jupierce**
@tdawson

### CODE REVIEW

**Comment by tdawson** [comment]
👍

File: jobs/build/ose/scripts/merge-and-build.sh [hunk 1]
```diff
@@ -4,6 +4,41 @@
 
 set -o xtrace
 
+# Checks the master and stage branch of the repo for the Version in the specified spec filename.
+# Calculates a higher version string which can be used with tito.
+function getPostStageVersion {
+    SPEC_FILE="$1"
+    if [ "$SPEC_FILE" == "" ]; then
```
Comment (jhadvig) [inline jobs/build/ose/scripts/merge-and-build.sh]: Idiomatic way should be `if [ -z "$SPEC_FILE" ]; then`

Comment (jupierce) [inline jobs/build/ose/scripts/merge-and-build.sh]: @jhadvig Old habits die hard! Fixed.

File: jobs/build/ose/scripts/merge-and-build.sh [hunk 2]
```diff
@@ -4,6 +4,41 @@
 
 set -o xtrace
 
+# Checks the master and stage branch of the repo for the Version in the specified spec filename.
+# Calculates a higher version string which can be used with tito.
+function getPostStageVersion {
```
Comment (jhadvig) [inline jobs/build/ose/scripts/merge-and-build.sh]: functions name should be lower-cased and with underscores to separate words.. so `get_post_stage_versions()`

Comment (jhadvig) [inline jobs/build/ose/scripts/merge-and-build.sh]: sorry for this additional nit... was going through the PR once again and find it... otherwise LGTM :+1:

**Pull request merged**